packages/contracts-bedrock/scripts/DeployOPChain.s.sol

```diff
@@ -145,6 +152,12 @@ contract DeployOPChainInput is BaseDeployIO {
         DeployUtils.assertImplementationSet(address(_opcmProxy));
         return _opcmProxy;
     }
+
+    function saltMixer() public view returns (string memory) {
+        // TODO: Temporarily allowing empty 'saltMixer' to be provided.
+        // require(bytes(_saltMixer).length != 0, "DeployOPChainInput: not set");
```

Comment:
Any reason not to permanently allow this? It seems ok to me if saltMixer is empty

Reply:
https://github.com/ethereum-optimism/optimism/pull/12152/commits/012cdf54a9504bd6b62f9e6eab4acce452642193